lib/ui/painting.dart

```diff
@@ -99,9 +99,26 @@ class Color {
   /// For example, to get a fully opaque orange, you would use `const
   /// Color(0xFFFF9000)` (`FF` for the alpha, `FF` for the red, `90` for the
   /// green, and `00` for the blue).
-  const Color(int value) : value = value & 0xFFFFFFFF;
-
-  /// Construct a color from the lower 8 bits of four integers.
+  const Color(int value)
+      : a = ((0xff000000 & value) >> 24) / 255.0,
+        r = ((0x00ff0000 & value) >> 16) / 255.0,
+        g = ((0x0000ff00 & value) >> 8) / 255.0,
+        b = ((0x000000ff & value) >> 0) / 255.0,
+        colorSpace = ColorSpace.sRGB;
+
+  /// Construct a color with normalized color components.
+  const Color.from(
```

Comment:
I feel like this deserves a bit more documentation and links from other constructors. For starters: "why would I want to use this constructor over other constructors"?

Reply:
Also link to color space enum.

Reply:
Done.